pandas/plotting/_core.py

```diff
@@ -864,6 +864,21 @@ def line(self, x=None, y=None, **kwargs):
             The values to be plotted.
             Either the location or the label of the columns to be used.
             By default, it will use the remaining DataFrame numeric columns.
+        color : str, int, array_like, or dict, optional
+            The color of each line for each row. Possible values are:
```

Comment:
Is `row` just a variable on the `x-axis`? If so may be better to specify as such; I think ambiguous otherwise

Reply:
Thanks for your review - what I meant was for "for each of the DataFrame's rows" (as that's how the bars are grouped). Would that be clear enough?

Reply:
Aren't the columns the things that get different colors though? Can you maybe post a screenshot of an example to discuss?

Reply:
Yes - what I was trying to say (albeit somewhat unclearly) was that this process is repeated for each row.

I've reworded it to just mention columns, it's probably clearer this way